ballerina/json_api.bal

```diff
@@ -53,10 +53,26 @@ public isolated function parseStream(stream<byte[], error?> s, Options options =
 
 # Converts a value of type `anydata` to `json`.
```

Comment:
I think we should improve this doc comment. Plese refer to the documentation of `toJson` method of lang

Reply:
+1 , FYI @SasinduDilshara

Reply:
Created a issue to track this
https://github.com/ballerina-platform/ballerina-library/issues/7005